docs/getting-started/install-endpoint.asciidoc

```diff
@@ -24,25 +24,25 @@ If you're using the Elastic Agent on macOS Mojave (10.14) or later, ensure that
 [role="screenshot"]
 image::images/install-endpoint/security-integration.png[]
 +
-2. On the Administration page of the security app or the Elastic Endpoint Security integration page in Ingest Manager, select **Add Endpoint Security**. The integration configuration page appears.
-3. Select a configuration for the Elastic Agent. You can use either the **Default config**, or adds security integration to a custom or existing configuration. For more details on Elastic Agent configuration settings, see {ingest-guide}/elastic-agent-configuration.html[Configuration settings].
-4. Configure the Elastic Endpoint Security integration with a name and optional description. When done configuring, select **Save integration**. Kibana redirects you back to the administration section of the security app.
+2. On the Administration page of the {security-app} or the Elastic Endpoint Security integration page in {fleet}, select **Add Endpoint Security**. The integration configuration page appears.
```

Comment:
I would remove the `{ }` around fleet for now.

Reply:
As far as I can tell, the fleet redirects are not in place yet. I'll loop back around to this in my Onboarding updates PR.